### ISSUE TITLE
Snes 8:7 aspect ratio option not working

### ISSUE THREAD

**Issue by sekuza009** [dragoonDorise/EmuDeck#377]
Currently the way the option reads in the Expert Mode installation of Emudeck states that if you use the checkbox it will use the 8:7 option, but it actually installs using 4:3. I assume this is something that's easy enough to fix just by changing the description.

Edit: Actually upon messing with it, it doesn't seem like the setting does anything regardless of setting. Always seems to be 4:3.

**Comment by notimp** [comment]
Yep, I had this issue as well. The issue is, that the bezel configuration overrides the core configuration, on aspect ratio.

Its a real issue.

To fix it for yourself (before Emudeck does), open snes.cfg in ~/.var/apps/org.libretro.RetroArch/config/retroarch/config/snes9x/ in a texteditor

~/ == home
/.var/ == hidden folder, make sure you enable show hidden folders in linux file explorer first

And search fore aspect ratio index - it should be set to 20 or 21. I believe its 21. Change that to one minus one (so 20), or plus one (so 22), save, and see if it worked. :)

(I believe its 21 and you need to change it to 20, but its from memory - and I'm sure, that changing it by 1 (unsure which direction ;) ) fixed the issue for me. :) Sorry for being so vague. :)

Issue in code is this: aspect_ratio_index needs to be set for 4:3 as well (to core provided), otherwise you will be stuck with 8:7

```
RetroArch_snes_ar43(){
	RetroArch_setOverride 'snes.cfg' 'Snes9x'  'input_overlay_scale_landscape' '"1.170000"'
	RetroArch_setOverride 'snes.cfg' 'Snes9x'  'input_overlay_aspect_adjust_landscape' '"0"'
	RetroArch_setOverride 'snes.cfg' 'Snes9x'  'input_overlay' '"~/.var/app/org.libretro.RetroArch/config/retroarch/overlays/pegasus/snes.cfg"'
}

RetroArch_snes_ar87(){
	RetroArch_setOverride 'snes.cfg' 'Snes9x'  'input_overlay' '"~/.var/app/org.libretro.RetroArch/config/retroarch/overlays/pegasus/snes87.cfg"'
	RetroArch_setOverride 'snes.cfg' 'Snes9x'  'input_overlay_scale_landscape' '"1.380000"'
	RetroArch_setOverride 'snes.cfg' 'Snes9x'  'input_overlay_aspect_adjust_landscape' '"-0.170000"'
	RetroArch_setOverride 'snes.cfg' 'Snes9x'  'aspect_ratio_index' '"21"'
}
```

**Comment by notimp** [comment]
Edit: found it. :)

>aspect_ratio_index = " " where 0: 4:3 1: 16:9 2: 16:10 3: 16:15 4: 1:1 5: 2:1 6: 3:2 7: 3:4 8: 4:1 9: 4:4 10: 5:4 11: 6:5 12: 7:9 13: 8:3 14: 8:7 15: 19:12 16: 19:14 17: 30:17 18: 32:9 19: config (video_aspect_ratio setting) 20: 10:9 (1:1 PAR) 21: Core Provided 22: Custom

src: https://old.reddit.com/r/RetroPie/comments/500zz6/how_do_you_make_retroarch_respect_the_aspect/

So it is 20 (1:1 PAR) and it needs to be changed to 21 (Core provided)

**Comment by Godsbane** [comment]
@dragoonDorise can you take a look at the logic here?

**Comment by dragoonDorise** [comment]
The proper AR is 15, to make sure 8:7 is forced. I've also fixed a bug with the bezels not being adjusted for the 8:7 AR for the next release

**Comment by dragoonDorise** [comment]
This should be fixed in 2.0.8. Please reopen this if you still find issues!
